scripts/run.sh

```diff
@@ -200,6 +205,8 @@ fi
       --runType batch
       --skipNEvents ${SKIP_N_EVENTS}
       --hepmc3.useHepMC3 ${USEHEPMC3:-true}
+      --physics.alternativeStableStatuses "{$((${SIGNAL_STATUS:-0}+1)), $((${BG1_STATUS:-0}+1)), $((${BG2_STATUS:-0}+1)), $((${BG3_STATUS:-0}+1)), $((${BG4_STATUS:-0}+1))}"
```

Comment:
please use the format:  --physics.alternativeStableStatuses="201 2001"

Reply:
Agreed. The format for other vectors on the ddsim/npsim command line is e.g. `ddsim --compactFile $DETECTOR_PATH/$DETECTOR_CONFIG.xml -G -N1 --gun.direction="0 1 0"`, and the similar notation with `{0, 1, 0}` segfaults. Since this command is likely to be used along with ddsim/npsim, it makes sense to stick to similar semantics.